<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
    <item>
     <widget class="QGroupBox" name="groupbox_displays">
      <property name="title">
       <string>Displays</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>按钮</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupbox_mainwindow">
      <property name="title">
       <string>Main window</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,2">
       <item>
        <widget class="QGroupBox" name="groupbox_vis3d">
         <property name="title">
          <string>Vis3D</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabwidget_main">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>输出</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTextEdit" name="textedit_log_info"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabwidget_setting">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>主页</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_theme">
    <property name="title">
     <string>主题</string>
    </property>
    <addaction name="actionDefault"/>
    <addaction name="actionDark"/>
    <addaction name="actionDark_red"/>
    <addaction name="actionDark_purple"/>
    <addaction name="actionDark_lightgreen"/>
    <addaction name="actionDark_amber"/>
    <addaction name="actionLight_blue"/>
    <addaction name="actionLight_cyan"/>
    <addaction name="actionLight_lightgreen"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_theme"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDefault">
   <property name="text">
    <string>default</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>dark_teal</string>
   </property>
  </action>
  <action name="actionDark_red">
   <property name="text">
    <string>dark_red</string>
   </property>
  </action>
  <action name="actionDark_purple">
   <property name="text">
    <string>dark_purple</string>
   </property>
  </action>
  <action name="actionDark_lightgreen">
   <property name="text">
    <string>dark_lightgreen</string>
   </property>
  </action>
  <action name="actionDark_amber">
   <property name="text">
    <string>dark_amber</string>
   </property>
  </action>
  <action name="actionLight_blue">
   <property name="text">
    <string>light_blue</string>
   </property>
  </action>
  <action name="actionLight_cyan">
   <property name="text">
    <string>light_cyan</string>
   </property>
  </action>
  <action name="actionLight_lightgreen">
   <property name="text">
    <string>light_lightgreen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
